### ISSUE TITLE
What things should be in a BEAST catalog paper?

### ISSUE THREAD

**Issue by lea-hagen** [BEAST-Fitting/beast#333]
Let's use this to compile things that we might want to have in catalog papers.

**Comment by lea-hagen** [comment]
* comparing fits with spectrally typed stars
* candidate extinguished early type stars

**Comment by karllark** [comment]
- spatial distributions of different types of stars
- candidate dusty AGB stars (via "bug")

**Comment by karllark** [comment]
- basic details of the fits, accuracy
- HR diagram
- correlation plots between parameters

**Comment by karllark** [comment]
- sensitivity tests using simulations with the AST derived noise model to say how accurate the measurement of the star/dust parameters

**Comment by karllark** [comment]
- listing of "bad" fits (via chisqr cut?) as candidate non-single stars in the galaxy (foreground, background AGN, binaries in the galaxy, etc.)

**Comment by lea-hagen** [comment]
* Naive maps of parameters (A_V, R_V)
* completeness (e.g., for a given stellar mass, how far in A_V you can see it)

**Comment by lea-hagen** [comment]
* histograms of best-fit parameters
* naive IMF (could do completeness corrections too)

**Comment by mfouesneau** [comment]
I think the catalog paper should be only validating the results, not deeply be science. The latter should be in individual papers for more impact.

Maybe give the guidelines on how to do some science cases if you feel it should be there.

**Comment by karllark** [comment]
I agree that a catalog paper should focus on the catalog and validating it.  But some science is good to motivate the use of the catalog.

Of course at this point - I would support any BEAST paper whatever the person writing it wants to have in it.  ;-)

**Comment by mfouesneau** [comment]
I give you my experience in writing validation papers... Science aspects are completely overlooked and forgotten. 
I understand the "energy" pool to write these papers of course.

**Comment by lea-hagen** [comment]
* Especially for the Magellanic Clouds, where there are multiple distances in the grid, make a CMD and/or HR diagram with the points colored by distance precision.  This would be a first step towards evaluating the degree to which the MegaBEAST will be able to constrain the distance.

**Comment by lea-hagen** [comment]
* Something (plots and/or numbers) to show that the noise model isn't changing drastically between SD/background bins.  This will help evaluate whether our bins are narrow enough.

**Comment by karllark** [comment]
> * Something (plots and/or numbers) to show that the noise model isn't changing drastically between SD/background bins.  This will help evaluate whether our bins are narrow enough.

See #398 for more discussion on this topic.

**Comment by lea-hagen** [comment]
* plots of 2D PDFs (see discussion in #303)
